packages/client/src/runtime/core/errorRendering/applyValidationError.test.ts

```diff
@@ -2329,10 +2329,71 @@ describe('Union', () => {
               }
             }
 
-            Argument \`<bold>gt</intensity>\`: Invalid value provided. Expected <green>String</color> or <green>StringFilter</color>, provided <red>Int</color>.
+            Argument \`<bold>email</intensity>\`: Invalid value provided. Expected <green>String</color> or <green>StringFilter</color>, provided <red>Int</color>.
 
         `)
   })
+
+  test('longest path: InvalidArgumentType vs RequiredArgumentMissing', () => {
```

Comment:
Here it's actually not about the longest path? But about `InvalidArgumentType` score?